reports.js

```diff
@@ -0,0 +1,90 @@
+function initChart() {
+    const chart = new CanvasJS.Chart("lineChart", _getOptions())
+    chart.render();
+
+    function _getOptions() {
+        return {
+            legend: {
+
+                itemclick: function (e) {
+                    console.log(e.dataSeries.legendText = "clicked")
+                    chart.render();
+                }
+
+            }, 
+            backgroundColor: "rgba(230, 230, 250, 0.4)",
+            title: _getTitle(),
+            data: selectedCoins.map((coin) => _getData("line", coin))
+        }
+
+        function _getTitle() {
+            return { text: "Currency Exchange - Updated Live !" }
+        }
+
+        function _getData(plotType, coin) {
+            return {
+                showInLegend: true,
+                legendText: coin.symbol,
+                type: plotType,
+                dataPoints: coin.history
+            }
+        }
+    }
+
+}
+
+async function updateChart() {
+
+    let coinSymbols = selectedCoins.map((coin) => coin.symbol).join(",");
+
+    const url = `https://min-api.cryptocompare.com/data/pricemulti?fsyms=${coinSymbols}&tsyms=USD`
+    const result = await fetch(url);
+    const jsonResult = await result.json();
+
+    const errorMessage = "Error";
+    if (jsonResult.Response === errorMessage) {
+        DOM.content.innerHTML = "";
+        const swalTitleText = "We're Sorry :(";
+        const swalBodyText = "The Selected Coin is Not a Valid coin that corresponds with our Third Party API...You can close this pop-up and return to our Home Page to select a new one";
+        const swalIconType = "error"
+        swal(swalTitleText, swalBodyText, swalIconType, {
+        }).then(() => getCryptoCoinsInit())
```

Comment:
you are repeating the same sweetAlert functionality instead creating a function and reuse